policyengine_canada/parameters/gov/provinces/mb/credits/basic_personal_amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: Manitoba provides the following basic personal amount. 
+values:
+  2022-01-01: 10_145
+  2023-01-01: 15_000
+metadata:
+  unit: currency-CAD
+  label: Manitoba Basic Personal Amount
+  reference:
+  - title: 2022 Manitoba, Personal Tax Credits Return, TD1MB
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1mb/td1mb-22e.pdf #page=1
+  - title: 2023 Manitoba, Personal Tax Credits Return, TD1MB
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1mb/td1mb-23e.pdf #page=1
```

Comment:
add the reference to the file: https://web2.gov.mb.ca/laws/statutes/archive/i010(2020-11-05)e.php?df=2019-06-03 (point to section 4.6)

Reply:
Shall I replace by this file https://web2.gov.mb.ca/laws/statutes/ccsm/_pdf.php?cap=i10, it's the most recent file I can find. The basic personal amount is still at section 4.6.

Reply:
we can use both of the references